src/tls13.c

```diff
@@ -7083,8 +7469,22 @@ static int SendTls13Finished(WOLFSSL* ssl)
             ssl->serverFinished_len = finishedSz;
         }
     #endif
+
+#ifdef WOLFSSL_DTLS13
+    if (ssl->options.dtls) {
+        dtlsRet = dtls13_handshake_send(ssl, output, outputSz,
+                                        dtls13_get_rl_header_length(ssl, 1)
+                                        + headerSz + finishedSz,
+                                        finished, 1);
+        if (dtlsRet != 0 && dtlsRet != WANT_WRITE)
+            return ret;
+
+        goto derive_key;
```

Comment:
Don't use goto like this.
#ifdef WOLFSSL_DTLS13
    if (ssl->options.dtls) {
    }
    else
#endif
    {
    }